.azure-pipelines/dotnet.yml

```diff
@@ -11,7 +11,8 @@ trigger:
       - hotfix/*
   paths:
     include:
-      - src/dotnet/Azure.ClientSdk.Analyzers
+      - dotnet.proj
+      - src/dotnet/*
```

Comment:
I don't think /* is doing what you expect as devops doesn't support wildcards in the paths.

Reply:
Oh I see. Will `src/dotnet` work?

Reply:
Yes I believe it should work without the *